fiftyone/utils/openimages.py

```diff
@@ -1559,28 +1559,24 @@ def _download_specific_images(
             len(inputs),
         )
 
-    global s3_client
-    s3_client = None
-
     with fou.ProgressBar(total=len(inputs)) as pb:
-        with multiprocessing.Pool(num_workers) as pool:
+        with multiprocessing.Pool(num_workers, _initialize_worker) as pool:
             for _ in pool.imap_unordered(_do_s3_download, inputs):
                 pb.update()
 
 
-def _initialize_pool():
+def _initialize_worker():
     global s3_client
-    if s3_client:
-        return
     s3_client = boto3.client(
         "s3",
         config=botocore.config.Config(signature_version=botocore.UNSIGNED),
     )
 
 
 def _do_s3_download(args):
-    _initialize_pool()
     filepath, filepath_download = args
+
+    global s3_client
```

Comment:
This can be removed, you only need to use the global keyword when assigning the variable. https://stackoverflow.com/questions/9366212/correct-use-of-global-variables-in-python-3